### DISCUSSION TITLE
Python module exifread missing

### DISCUSSION THREAD

**Discussion by GeneWilburn** [Homebrew/discussions#5618] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
% brew search exifread
==> Formulae
exiftran

==> Casks
exifrenamer
```

### Output of `brew doctor`

```shell
None
```

### Description of issue

Is there a way to add the Python module exifread? I use exifread in a Linux Python script and I'd like to use it with Homebrew on my Mac.

Apologies if this is not the right way to make a request. I really use Homebrew a lot and would love to add this module.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew
